lib/config/default.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     upgradeInsecureRequests: 'auto',
     reportURI: undefined
   },
+  cookiePolicy: 'strict',
```

Comment:
After reading my own commit we might want to go for lax by default. otherwise people might run CodiMD locally and end up with  non functional setups leaving a bad impression.

https://github.com/codimd/server/commit/3d1fab0512a828927f277796a0e23e8133230be5

Reply:
Done in 387e6682